conftest.py

```diff
@@ -8,3 +8,12 @@
 # avoid duplicating the implementation
 
 from cms.conftest import _django_clear_site_cache, pytest_configure  # pylint: disable=unused-import
+
+import pytest
+
+@pytest.fixture(autouse=True)
+def no_webpack_loader(monkeypatch):
+    monkeypatch.setattr(
+        "webpack_loader.templatetags.webpack_loader.render_bundle",
+        lambda x: ''
+    )
```

Comment:
This makes `render_bundle` noop in unittests. We need it because many of the unittests do full-page renders, meaning they hit `render_bundle`, which errors out when we haven't built assets. IMO python unittests should not have an asset pipeline dependency.